server/bundler/babel/babel-plugin-transform-wpcalypso-async/index.js

```diff
@@ -98,10 +98,11 @@ module.exports = ( { types: t } ) => {
 					// Transform to asynchronous require.ensure
 					path.replaceWith(
 						t.callExpression( t.memberExpression( t.identifier( 'require' ), t.identifier( 'ensure' ) ), [
-							argument,
+							t.arrayExpression( [ argument ] ),
```

Comment:
these are both moves to align with the new preferred require.ensure calling convention